benchmarks/htap/__init__.py

```diff
@@ -19,19 +23,34 @@ def add_parser(subparsers):
         '--olap-timeout', default=900, type=int, help=(
         'Timeout for OLAP queries in seconds, default: 900'))
 
+    parser.add_argument(
+        '--oltp-timeout', default=50, type=int, help=(
+        'Timeout for OLAP queries in milliseconds, default: 10'))
+
+    parser.add_argument(
+        '--tpcc-csv-interval', default=60, type=int, help=(
+        'How often to report tpcc stats to the csv file (for the latest 60 seconds)'))
```

Comment:
(for the latest 60 seconds) --> in seconds, default 60

Reply:
done